subsys/usb/usb_c/usbc_prl.c

```diff
@@ -1164,6 +1264,18 @@ static const struct smf_state prl_tx_states[PRL_TX_STATE_COUNT] = {
 		prl_tx_suspend_run,
 		NULL,
 		NULL),
+#ifdef CONFIG_USBC_CSM_SOURCE_ONLY
```

Comment:
Should the sink states also be config-guarded?

Reply:
done